src/cdk/schematics/ng-update/devkit-migration.ts

```diff
@@ -43,4 +43,6 @@ export abstract class DevkitMigration<Data> extends Migration<Data, DevkitContex
 }
 
 export type DevkitMigrationCtor<Data> = Constructor<DevkitMigration<Data>> &
-  {[m in keyof typeof DevkitMigration]: typeof DevkitMigration[m]};
+  {
+    [m in keyof typeof DevkitMigration]: typeof DevkitMigration[m];
+  };
```

Comment:
It looks like this is what is breaking the lint job. I don't see why the formatting on this changed since it's not a JSON file.

Reply:
potentially a prettier update, or the file was not formatted properly before (and didn't get checked)